yogstation/code/modules/mob/living/emote.dm

```diff
@@ -74,3 +74,4 @@
 		var/light_dab_speed = rand(3,7)
 		H.DabAnimation(angle = light_dab_angle , speed = light_dab_speed)
 		H.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
+		SSachievements.unlock_achievement(/datum/achievement/dab,H.client)
```

Comment:
No

Reply:
I do not control the rate at which spacemen dab